rllib/models/distributions.py

```diff
@@ -13,10 +13,24 @@ class Distribution(abc.ABC):
     """The base class for distribution over a random variable.
```

Comment:
This file explicitly skipped in the `BUILD` file. Could we remove it from the list?